### PR TITLE
Octave: version bumb 4.0.3 + FLTK

### PR DESCRIPTION
**Pull request by schoeps**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
I haven't fixed "C: 133: col 30: Prefer `to_s` over string interpolation."
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### Updates to existing formula: have you

- [x] Removed the `revision` line, if any, when bumping the version number?
- [x] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

File: octave.rb [hunk 1]
```diff
@@ -71,7 +77,6 @@ class Octave < Formula
 
   # options, disabled by default
   option "with-audio",             "Use the sndfile and portaudio libraries for audio operations"
-  option "with-fltk",              "Build with FLTK graphics backend"
```
Comment (dpo) [inline octave.rb]: Need to update the `build.with? "fltk"` below.

**Comment by dpo** [comment]
```
==> brew audit homebrew/science/octave
==> FAILED
Error: 1 problem in 1 formula
homebrew/science/octave:
  * 15: Trailing whitespace was found
```

File: octave.rb (hunk 1 above)
Comment (schoeps) [inline octave.rb]: Sorry, I don't get it. I made an optional "--with-fltk" switch recommended "--without-fltk", so the program logic does not need to be changed?!

Comment (dpo) [inline octave.rb]: Sorry I missed line 66.

**Comment by dpo** [comment]
Thank you!

**Pull request closed without merging**